opengemini-client-api/src/main/java/io/opengemini/client/api/RetentionPolicy.java

```diff
@@ -0,0 +1,24 @@
+package io.opengemini.client.api;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class RetentionPolicy {
```

Comment:
Is there any diff between RetentionPolicy and RpConfig？It seems that there is no code referring RetentionPolicy,